src/user_interface/main_page.py

```diff
@@ -129,7 +129,7 @@ def __init__(self, input_queue, output_queue):
         self.play_page = PlayingPage()
         self.play_page.nav_home.clicked.connect(self.go_to_home_page)
 
-        self.settings_page = SettingsPage()
+        self.settings_page = SettingsPage(self.comm_system)
```

Comment:
You shouldn't have to pass the comm system in. The playing page can just create its own and itll get synced up